adoc/Release-Notes.adoc

```diff
@@ -21,6 +21,67 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_2_1']
+=== Release 1.2.1, September 2018
+
+[id='sec.1_2_1.issue']
+==== Known Issues
+
+* If you are upgrading to {product} 1.2.1, you will need to run the `helm upgrade` command with two additional parameters: `helm upgrade --force --recreate-pods`. This will be the case with all future 1.x upgrades due to the expected behaviour with Kubernetes that unready pods won’t get upgraded.
+
+* Similar to {caaspa} 3, {azure} uses an updated version of Kubernetes that mandates a stricter security policy via PodSecurityPolicy (PSP), which is included as part the {product} Deployment Guide. Any namespace tied to {product} requires privileged ports to be accessible needs to have to have a PSP set appropriately for access. This would include the default conventions of `scf`, `uaa`, `stratos-ui`, `mysql-sidecar` and `postgres-sidecar` as per our documentation.
+
+* {azure} users will need to add `--node-osdisk-size=60` as part of their AKS creation command to ensure the root partition has enough space for the install and future upgrades. The {product} Deployment Guide has a section called `Create Resource Group and AKS Instance` and the `node-osdisk-size` parameter would need to be added to the `az aks create` command.
```

Comment:
Could you add the expansion of the acronym "AKS" somewhere and again, do "1. issue, 2. solution"?

Suggestion:
"If you are using {azure}, ensure that the root partition has enough space for the installation and potential upgrades. To do so, add the parameter `--node-osdisk-size=60` to the command that creates the AKS (Azure Kubernetes Service) instance (`az aks create`). For more information about this parameter, see [insert link here]."